java/src/jmri/jmrit/operations/rollingstock/cars/CarManager.java

```diff
@@ -458,6 +458,20 @@ public static String calculateCarWeight(String carLength) throws NumberFormatExc
         nf.setMaximumFractionDigits(1);
         return nf.format(doubleCarWeight); // car weight in ounces.
     }
+    
+    /**
+     * Used to determine is any car has been assigned a division
```

Comment:
No need to change. But it seems to me that the word "is" should be "if".

Reply:
you're correct